### DISCUSSION TITLE
Installation Trouble: [E049] Can't find spaCy data directory: 'None'.

### DISCUSSION THREAD

**Discussion by sharmaapeksha960** [explosion/spaCy#10329] [Help: Installation] [answered]
Hi All,

I am very new to this so would really need any guidance on help on below issue:

Background: 
I have installed spacy and en_core_web_md as below:

_pip install spacy
pip install ...en_core_web_md-3.0.0.tar.gz_

All of these were executed fine

and then imported spacy and load language model. First line to import spacy worked fine **whereas for load language model generated below error**

_import spacy
nlp = spacy.load("en_core_web_md")_

**[E049] Can't find spaCy data directory: 'None'. Check your installation and permissions, or use spacy.util.set_data_path to customise the location if necessary.**

Can anyone help me what I am missing here?

**Comment by polm** [comment, marked as answer]
Sorry you're having trouble with this. 

You generally don't want to download pretrained pipelines directly, instead you should use the `spacy download` command to get the right version and get everything set up correctly. You can find the exact commands to install spaCy for your OS and environment on the [install page](https://spacy.io/usage). Can you try following the instructions there? If they don't work let us know. 

I'm not exactly sure why you're getting the error you have, but it may be due to some kind of version mismatch.

**Reply by sharmaapeksha960** [reply]
Thanks @polm 👍 

I just cleaned my environment and then restarted jupyter notebook and it worked!! :)
